Examples/Scripts/Python/full_chain_odd.py

```diff
@@ -318,6 +318,16 @@
             if args.ttbar
             else TruthSeedRanges()
         ),
+        initialSigmas=[
+            1 * u.mm,
+            1 * u.mm,
+            1 * u.degree,
+            1 * u.degree,
+            0.1 * u.e / u.GeV,
+            1 * u.ns,
+        ],
+        initialSigmaPtRel=0.1,
```

Comment:
How tuned is this 0.1? 
Does it matter to be linked to the expected Relative SigmaPt from the detector design? i.e. ITK and ODD should have rather different detector resolutions

Reply:
it is a random number that popped into my head. 10% pt resolution seemed big enough to me